requirements.txt

```diff
@@ -1 +1,5 @@
 ops
+lightkube
+lightkube-models
+pydantic
+pytest-interface-tester
```

Comment:
We need this in the charm?

Reply:
Yes, that is used by libs based on pydantic to validate the relation data.